crates/swc_ecma_transforms_compat/src/es2022/class_properties/mod.rs

```diff
@@ -58,10 +59,93 @@ pub struct Config {
 }
 
 struct ClassProperties {
+    extra: ClassExtra,
     c: Config,
     private: PrivateRecord,
 }
 
+struct ClassExtra {
+    lets: Vec<VarDeclarator>,
+    vars: Vec<VarDeclarator>,
+    stmts: Vec<Stmt>,
+}
+
+impl ClassExtra {
+    fn new() -> Self {
```

Comment:
Isn't `#[derive(Default)]` better?
Are you going to add a field that does not have a good default?

Reply:
No